src/components/Swap/Status/StatusDialog.tsx

```diff
@@ -20,7 +26,7 @@ const TransactionRow = styled(Row)`
   flex-direction: row-reverse;
 `
 
-type PendingTransaction = Transaction<SwapTransactionInfo | WrapTransactionInfo>
+type PendingTransaction = Transaction<SwapTransactionInfo | WrapTransactionInfo | UnwrapTransactionInfo>
```

Comment:
why can't this be `Transaction<TransactionInfo>`?

Reply:
NB: That's equivalent to just `Transaction` (as `TransactionInfo` defaults).